example-workflows/Technique_2_Slack_API_Method/README.md

```diff
@@ -0,0 +1,40 @@
+# Technique 2: Slack API method
+
+A bot token or user token or [token of some other kind][tokens] must be used to
```

Comment:
wait, so any kind of token is fair game?

Reply:
Yep! What'ever's passed in here is sent with the API method used 🔍